project/merge.go

```diff
@@ -50,8 +50,21 @@ func Merge(p *Project, bytes []byte) (map[string]*ServiceConfig, error) {
 		datas[name] = data
 	}
 
-	err = utils.Convert(datas, &configs)
-	return configs, err
+	if err := utils.Convert(datas, &configs); err != nil {
+		return nil, err
+	}
+
+	adjustValues(configs)
+	return configs, nil
+}
+
+func adjustValues(configs map[string]*ServiceConfig) {
+	// yaml parser turns "no" into "false" but that is not valid for a restart policy
+	for _, v := range configs {
+		if v.Restart == "false" {
+			v.Restart = "no"
+		}
+	}
```

Comment:
I don't think this is the direction we want to go. We've decided against it in `docker-compose` (at least for now).

Re-writing boolean values to string values is not really intuitive.  Could we instead warn the user that they have the wrong type?

I think this will just fix itself when #34 is completed. The jsonschema will raise a validation error if this field is a boolean